addons/api/addon/serializers/role.js

```diff
@@ -15,23 +15,32 @@ export default class RoleSerializer extends ApplicationSerializer {
    * serialized.
    * If `adapterOptions.principalIDs` is set (to an array of user and
    * group IDs), then the payload is serialized via `serializeWithPrincipals`.
+   * If `adapterOptions.grantScopeIDs` is set (to an array of scope IDs or
+   * keywords), then the payload is serialized via `serializeWithGrantScopeIDs`.
    * @override
    * @param {Snapshot} snapshot
    * @return {object}
    */
   serialize(snapshot) {
-    const grantStrings = get(snapshot, 'adapterOptions.grantStrings');
     let serialized = super.serialize(...arguments);
+
+    const grantStrings = get(snapshot, 'adapterOptions.grantStrings');
     if (grantStrings)
       serialized = this.serializeWithGrantStrings(snapshot, grantStrings);
+
     const principalIDs = snapshot?.adapterOptions?.principalIDs;
     if (principalIDs)
       serialized = this.serializeWithPrincipals(snapshot, principalIDs);
+
+    const grantScopeIDs = snapshot?.adapterOptions?.grantScopeIDs;
+    if (grantScopeIDs)
```

Comment:
I know this was how it was previously but can we use braces for if statements that aren't on the same line?

Reply:
Yep that is right. I will fix these.